src/config/dynamic-theme-fixes.config

```
@@ -6206,17 +6206,61 @@ gog.com
 
 INVERT
 i.icn.icn--close
+svg.icon-wrapper-icon
+svg.big-spot__add-to-cart-icon
+svg.ic-svg:not(.button__icon):not(.productcard-slider__nav-icon)
+.productcard-thumbnails-slider-pagination
+.carousel-pagination__page
+.big-spot__carousel-pages-container
+.big-spots__arrow-icon
+.discover-games-more__icon
 
 CSS
 .product-tile__info:hover {
     background-color: var(--darkreader-neutral-background) !important;
 }
+a.product-tile__content {
+    background-color: var(--darkreader-neutral-background) !important;
+}
+.explore-galaxy {
+    border-top: 0 !important;
+}
+.galaxy-tooltip__layer {
+    background: ${#d9d9d9} !important;
+    color: var(--darkreader-neutral-text) !important;
+}
+.big-spot__content, .big-spot__text, .big-spot__action {
+    backdrop-filter: brightness(50%) !important;
+}
+.big-spot__title, .big-spot__super-title, .big-spot__action {
+    filter: brightness(130%) !important;
+}
+.wishlist-button, .review__read-more, .review-new__add-button, .review-new__guideline-button {
+    background: none !important;
```

```suggestion
    background-image: none !important;
```
Let's specify it a bit further.